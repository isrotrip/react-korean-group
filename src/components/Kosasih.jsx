import React, { Component } from 'react'

export default class Kosasih extends Component {
  render() {
    return (
      <div>
          <h1>Halo kosasih <span>😘🥰😍😍</span></h1>
      </div>
    )
  }
}
