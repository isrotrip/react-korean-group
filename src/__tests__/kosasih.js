import React from 'react'
import Kosasih from '../components/Kosasih'
import {render} from 'react-testing-library'

test('it will works with snapshot', () => {
  const {container} = render(<Kosasih />)

  expect(container).toMatchSnapshot()
})