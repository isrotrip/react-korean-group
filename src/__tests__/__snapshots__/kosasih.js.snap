// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it will works with snapshot 1`] = `
<div>
  <div>
    <h1>
      Halo kosasih 
      <span>
        😘🥰😍😍
      </span>
    </h1>
  </div>
</div>
`;
